test-data/unit/plugins/add_classmethod.py

```diff
@@ -7,7 +9,7 @@
 
 
 class ClassMethodPlugin(Plugin):
-    def get_base_class_hook(self, fullname: str) -> Optional[Callable[[ClassDefContext], None]]:
+    def get_base_class_hook(self, fullname: str) -> Callable[[ClassDefContext], None] | None:
```

Comment:
Speaking of this change - I've just enabled ruff's "pyupgrade" (#16023) which should keep enforcing this, maybe change pyproject.toml to include the plugin?

Reply:
I will in later PRs, because right now - I don't know how to force `ruff` to include a subdirectory. Docs do not say much, so I will have to dig into the source code :)